Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/test/java/org/apromore/plugin/portal/processdiscoverer/PDAnalystTest.java

```diff
@@ -39,6 +39,9 @@
 import org.junit.runner.RunWith;
 import org.mockito.Mockito;
 import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.*;

```

Comment:
Same minor code style issue:
Its best to have specific imports than the wildcard * (https://checkstyle.sourceforge.io/config_imports.html#AvoidStarImport)

You can configure this in the IDE, so when you organise imports it only imports the necessary ones

Reply:
IntelliJ has this default config: more than 5 imports from the same package it will use wildcard.

![image](https://user-images.githubusercontent.com/12888841/149848489-ab8ea6f0-4f11-48cb-9665-16870ac82178.png)